src/compute-client/src/protocol/command.rs

```diff
@@ -73,7 +73,10 @@ pub enum ComputeCommand<T = mz_repr::Timestamp> {
     /// dataflows according to the given [`LoggingConfig`].
     ///
     /// [Creation Stage]: super#creation-stage
-    CreateInstance(LoggingConfig),
+    CreateInstance {
+        logging_config: LoggingConfig,
+        variable_length_row_encoding: bool,
+    },
```

Comment:
How about we introduce a new type `InstanceConfig` to wrap these two fields and then have a `CreateInstance(InstanceConfig)` command? I think that would simplify that replica code a bit, particularly the "has the instance config changed?" part, and even more so if we add more config options in the future.